aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1218,6 +1374,15 @@ module aptos_framework::stake {
         );
     }
 
+    /// Assuming we are in a middle of a reconfiguration (no matter it is immediate or async), get its start time.
+    fun get_reconfig_start_time_secs(): u64 {
+        if (reconfiguration_state::is_initialized()) {
+            reconfiguration_state::start_time_secs()
```

Comment:
is it possible for this to be initialized but not used (thus abort)?

Reply:
yes, in genesis. Fixed in #11721 at caller side.